extensions/dbt-documentation/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "dbt-documentation",
+  "title": "dbt Documentation Search",
+  "description": "Search the dbt documentation website https://docs.getdbt.com/",
+  "icon": "dbt-icon.png",
+  "author": "b-per",
+  "license": "MIT",
+  "categories": ["Documentation"],
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search dbt Documentation",
```

Comment:
nitpicking here, but the command together with the extension name look a bit verbose. I would suggest maybe call the extension title "dbt" and then the command title "Search Documentation" with the subtitle "dbt".

Reply:
Completely agree, the suggestion on the command title/subtitle is great and I just changed it.

In regards to the extension name, there is [already one](https://www.raycast.com/zsombor-flds/dbtcloud) related to dbt but focused on the commercial product dbt Cloud, so I thought it'd still make sense to mention that this is about the docs search for dbt.

Reply:
Ah ok, I didn't check. In that case I guess "dbt Documentation" as the name is fine.